app/views/application_records/show.haml

```diff
@@ -1,15 +1,16 @@
-%h1 Application Record
+%h1.h2="#{@record.position.name} Application Record"
```

Comment:
As in, the idea is that you would normally use an `h1` tag, but if you want your text to be inline, you use `.h1`, but there's no reason to use both, and definitely no reason to use an `h1` tag and the `.h2` class, which is a nice trick, but doesn't really communicate what you want.